README.md

```diff
@@ -319,41 +319,21 @@ module.exports = {
 };
 ```
 
-The `options` object contains the following:
-
-```tsx
-export interface TsdxOptions {
-  // path to file
-  input: string;
-  // Name of package
-  name: string;
-  // JS target
-  target: 'node' | 'browser';
-  // Module format
-  format: 'cjs' | 'umd' | 'esm' | 'system';
-  // Environment
-  env: 'development' | 'production';
-  // Path to tsconfig file
-  tsconfig?: string;
-  // Is error extraction running?
-  extractErrors?: boolean;
-  // Is minifying?
-  minify?: boolean;
-  // Is this the very first rollup config (and thus should one-off metadata be extracted)?
-  writeMeta?: boolean;
-  // Only transpile, do not type check (makes compilation faster)
-  transpileOnly?: boolean;
-}
-```
+The `options` object type definition in [src/externalTypes.ts](src/externalTypes.ts).
 
 #### Example: Adding Postcss
 
 ```js
+const tsdx = require('tsdx');
```

Comment:
So I recently found out that JSDoc actually [supports imports](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#import-types) after seeing [some usage of it](https://github.com/developit/rollup-plugin-preserve-shebang/blob/cc98df81e83bb19d5e6e469bce77a93f577da045/index.mjs#L7), so we should be able to remove this entirely that way, and perhaps remove `package.json#main` as well
```suggestion
```